libc/src/__support/UInt.h

```diff
@@ -121,6 +121,7 @@ template <size_t Bits, bool Signed> struct BigInt {
         return lo;
       }
     } else {
+      // TODO: silence shift warning
```

Comment:
I guess Bits could be a value like 65, hence the warning?  Maybe

```c
if constexpr (Bits == 32) {
  return static_cast<T>((static_cast<T>(val[1]) << 64) + lo);
} else {
  ...
```
would silence the warning?

Reply:
then perhaps a pragma here disabling the warning is the best approach?

I guess the static_assert on line 30 would allow values like 128 or 256?

Reply:
I'm going to leave fixing this warning for a followup patch.